app/code/Magento/Catalog/Model/Product/Attribute/Source/StockStatus.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model\Product\Attribute\Source;
+
+use Magento\Eav\Model\Entity\Attribute\Source\AbstractSource;
+use Magento\Eav\Model\Entity\Attribute\Source\SourceInterface;
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Product stock status functionality model
+ *
+ */
+class StockStatus extends AbstractSource implements SourceInterface, OptionSourceInterface
+{
+    /**#@+
```

Comment:
Thanks @orlangur ! Sorry @XxXgeoXxX , I gave you a wrong suggestion, "/**#@+" is not supported by PHPDocumentor anymore as was correctly mentioned by @orlangur . Here is the source: https://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#DocBlock-templates

Considering the above, each constant should have it's own phpdoc